remoting/tests/pom.xml

```diff
@@ -82,5 +82,10 @@
             <type>pom</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
-1 to mocks like this.

The subsystem test framework allows you to create a config you want and execute management ops against it. There should be no need for mocks.